openstates/importers/organizations.py

```diff
@@ -14,7 +14,10 @@ def limit_spec(self, spec: _JsonDict) -> _JsonDict:
 
         name = spec.pop("name", None)
         if name:
+            # __icontains doesn't work for JSONField ArrayField
+            # so other_names follows "title" naming pattern
+            other_names = name.title().replace(" On ", " on ")
```

Comment:
This variable name confused me a bit, I'd vote for something like `other_name_lowercase_on` or something... since the variable `other_names` is plural, I expected it to be a list and was thrown by that.

Reply:
That is true, thanks!